### ISSUE TITLE
IX bid adapter not passing back width and height

### ISSUE THREAD

**Issue by dbirkbeck** [prebid/prebid-server#1903]
Hi.

The bid responses from the IX bid adapter do not contain a height and width property for banner ads. (Not sure about other ad types.) This causes issues with the renderAd function in Prebid when setRenderSize is called.

As a result of the missing height and width, I am seeing improper iframe width and height values being set from previous ad units which is cutting off ads. (My apologies if the following is hard to follow, it is difficult to explain.)

For example, I have 2 ad units. The first is 160x600 and the second is 300x250. I am seeing the 300x250 size ad being cut off because setRenderSize is being called with a width of 160 and a height of 600. (Values from the previous ad that was displayed in which valid width and height were provided.)

Manually setting the size of the rendered iframe AFTER renderAd is called (resetting the width and height to the proper values) works around this issue but it would be nice if it was fixed properly.

To reproduce, inspect the properties of the bid response from the IX bid request.  You will see no width or height properties whereas other adapters have them.

Thanks!

**Comment by SyntaxNode** [comment]
I'll bring this to the attention of IX.

**Comment by dbirkbeck** [comment]
> I'll bring this to the attention of IX.

Thank you!

**Comment by grossjo** [comment]
Hi,
We (IX) are looking into this. 

Thanks for bring this to our attention.

**Comment by grossjo** [comment]
@dbirkbeck  While I dig into this. Do you have a sample request/response or even a test page?

**Comment by dbirkbeck** [comment]
Hi. I attached a redacted screenshot from chrome's developer tools. As you can see, the top bid response (from another bid provider has width and height (h & w properties). The bottom bid response is from IX and does not contain the h & w properties. We have since put in place a work around for this issue so I don't have a screenshot of the broken ad placement unfortunately.

![IX_height_width](https://user-images.githubusercontent.com/22990147/123867528-b7f2f800-d8eb-11eb-910a-43f6a92f38ff.png)

**Comment by grossjo** [comment]
@dbirkbeck  We internally were able to get W/H responses from prebid server.  

This seems like it may be a configuration issue between the prebid server instance you are using and IX.
If you can let me know someone to reach out too, we can hopefully get this sorted quite easily.  As we probably need a bit more information about how PBS is configured in your use case.

**Comment by dbirkbeck** [comment]
@grossjo  We are using AppNexus' hosted Prebid server. See here:
https://prebid.adnxs.com/

**Comment by grossjo** [comment]
Just so I have all information, which publisher are you?
This may not be a setting  Appnexus controls inside their PBS, but a setting IX controls based on your pub/site id.

I'm also in the prebid.org slack group if you'd like to move this to a more private chat to share these details not in a public group.

**Comment by dbirkbeck** [comment]
Hi @grossjo. Unfortunately, I'm not a member as yet at prebid.org so I don't have access to slack. (Should probably look at changing that!) I am reluctant to post specific publisher/account details here. I may try to reach you through our IX publisher rep.
If you believe this is a configuration issue and not an adapter problem, I should close this and take things offline.

**Comment by grossjo** [comment]
I do think its a configuration issue, let's close this for now.  We can always reopen if I'm wrong.
Reach out to your IX Rep and ask them to reach out to me as well, and we can get this all sorted.

Cheers,
Joshua

**Comment by dbirkbeck** [comment]
Sounds good. Thx!
